### PR TITLE
Test case with keyword NONNC

### PR DESCRIPTION
**Pull request by tskille**
New test case using keyword NONNC. This keyword is currently not supported in OPM FLow.

### CODE REVIEW

File: editnnc/NONNC.DATA [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+------------------------------------------------------------------------------------------------
+
+--INCLUDE
+-- 'include/edit_nnc.inc' /
+
+-- repeted NNC, should be product of first and second entry
+--EDITNNC
+-- 3 1 3  4 1 1  0.0010 /
+-- 3 2 3  4 2 1  0.0010 /
+--/
```
Comment (bska) [inline editnnc/NONNC.DATA:159]: Do we expect to enable this `EDITNNC` specification at some point?  If not, maybe we could just remove these lines?

**Review by bska — COMMENTED** [review]
This looks good to me except possibly for the disabled `EDITNNC` data.  If we're going to reenable `EDITNNC` despite the `NONNC` keyword, then we should keep it, otherwise I think we should just remove the `EDITNNC` data as it might be confusing here.

File: editnnc/NONNC.DATA (hunk 1 above)
Comment (tskille) [reply editnnc/NONNC.DATA]: Thanks, I have removed this. I did not complete cleaning up the deck before sending this in.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**